juju-tutorials/how-to-deploy-openstack-on-openstack.md

```diff
@@ -184,7 +184,7 @@ Don't forget to get credentials for the new OpenStack. (e.g. `source openrc`, wh
       PROJECT_ID=$(openstack project list -f value -c ID --domain admin_domain)
       openstack quota set --instances 40 --secgroups 40 $PROJECT_ID
       ```
-   1. Keypair - allow ssh access to instances using a customer ssh public key
+   1. Keypair - allow ssh access to instance using a customer ssh public key
```

Comment:
This can be left as it was. It is a generic description, meaning that any instance (in plural) can benefit from the keypair creation.